packages/interface/src/api/activity/endpoint/apiAct007.ts

```diff
@@ -0,0 +1,73 @@
+import { HttpStatusCode } from "axios";
+
+import { z } from "zod";
+
+import { ActivityTypeEnum } from "@sparcs-clubs/interface/common/enum/activity.enum";
+
+/**
+ * @version v0.1
+ * @description 활동보고서의 활동을 수정합니다.(가동아리 동아리 신규등록 신청을 위한 예외적 활동보고서 작성 기능입니다.)
+ */
+
+const url = () => `/student/activities/activity/provisional`;
+const method = "POST";
+
+const requestParam = z.object({});
+
+const requestQuery = z.object({});
+
+const requestBody = z.object({
+  clubId: z.coerce.number().int().min(1),
+  name: z.coerce.string().max(255),
+  activityTypeEnumId: z.nativeEnum(ActivityTypeEnum),
+  durations: z.array(
+    z.object({
+      startTerm: z.coerce.date(),
+      endTerm: z.coerce.date(),
+    }),
+  ),
+  location: z.coerce.string().max(255),
+  purpose: z.coerce.string(),
+  detail: z.coerce.string(),
+  evidence: z.coerce.string(),
+  evidenceFiles: z.array(
+    z.object({
+      uid: z.coerce.string().max(255),
```

Comment:
```suggestion
      fileId: z.coerce.string().max(255),
```
❗ 파일 업로드 정책 변경으로 uid -> fileId로 변경해주세요. 늦게 전파해서 미안합니다..